src/cmd/MVP-4/main.go

```diff
@@ -110,62 +88,54 @@ func main() {
 	})
 
 	go func(msgChannel chan models.MessageTarget) {
-		for packet := range transportControllerConfig.Dump {
-			id, values := packetParser.Decode(packet)
-			values = podConverter.Convert(values)
-			values = displayConverter.Convert(values)
-			logger.Update(values)
-			if idToType[id] == "data" {
-				dataTransfer.Update(packetFactory.NewPacketUpdate(id, packet, values))
+		for update := range updateChan {
+			logger.Update(update.Fields)
+			if idToType[update.ID] == "data" {
+				dataTransfer.Update(update)
 			} else {
-				messageTransfer.Broadcast(packetFactory.NewPacketUpdate(id, packet, values))
+				messageTransfer.Broadcast(update)
 			}
 		}
 	}(messageChannel)
 
 	go func() {
 		for order := range orderChannel {
-			order.Values = displayConverter.Revert(order.Values)
-			order.Values = podConverter.Revert(order.Values)
-			transportController.Write(idToIP[order.ID], packetParser.Encode(order.ID, order.Values))
+			log.Println(order)
+			if err := boardMux.Request(order); err != nil {
+				log.Printf("request failed: %s\n", err)
+			}
 		}
 	}()
 
 	httpServer := server.Server{Router: mux.NewRouter()}
-	httpServer.ServeData(os.Getenv("POD_DATA_ENDPOINT"), podData)
-	httpServer.ServeData(os.Getenv("ORDER_DATA_ENDPOINT"), orderData)
 
-	handle := websocket_handle.RunWSHandle(httpServer.Router, "/backend", map[string]chan models.MessageTarget{
+	httpServer.ServeData("/backend/"+os.Getenv("POD_DATA_ENDPOINT"), podData)
+	httpServer.ServeData("/backend/"+os.Getenv("ORDER_DATA_ENDPOINT"), orderData)
+
+	websocket_handle.RunWSHandle(httpServer.Router, "/backend", map[string]chan models.MessageTarget{
 		"podData/update":    dataTransferChannel,
 		"message/update":    messageChannel,
-		"order/update":      ordChannel,
+		"order/send":        ordChannel,
 		"connection/update": connectionChannel,
-		"logger/enable":     loggerChannel,
+		"logger":            loggerChannel,
 	})
 
 	path, _ := os.Getwd()
 	httpServer.FileServer("/", filepath.Join(path, "static"))
 
 	go httpServer.ListenAndServe(os.Getenv("SERVER_ADDR"))
 
-	//Test with clients "staticClient"
-	httpServerClient := server.Server{Router: mux.NewRouter()}
-	httpServerClient.FileServer("/", filepath.Join(path, "staticClient"))
-
-	go httpServerClient.ListenAndServe(os.Getenv("SERVER_CLIENT_ADDR"))
-
 	interrupt := make(chan os.Signal, 1)
 	signal.Notify(interrupt, os.Interrupt)
 
+	log.Println("backend running!")
 loop:
```

Comment:
Put loop inside function

Reply:
I don't see the use of putting this on a function, mainly because this will be removed before release. This is just for debugging and we should use a better method